static/js/components/Navbar.js

```diff
@@ -101,7 +101,8 @@ export default class Navbar extends React.Component {
               </div>
               <div className="link">
                 <Icon name="person" />
-                <Link to={`/learner/${profile.username}`} onClick={closeDrawer} >
+                <Link to={`/learner/${SETTINGS.user ? SETTINGS.user.username : profile.username}`}
```

Comment:
Can we change this to just `SETTINGS.user.username`? I don't think the user will see this code unless they are logged in